build: Add required CFLAGS for libpcap

This patch adds some flags to required to properly link libpcap within the
OpenWrt environment.

Signed-off-by: Jo-Philipp Wich <jo@mein.io>

--- a/pppd/Makefile.linux
+++ b/pppd/Makefile.linux
@@ -210,8 +210,8 @@ LIBS	+= -ldl
 endif
 
 ifdef FILTER
-LIBS    += -lpcap
-CFLAGS  += -DPPP_FILTER
+LIBS    += -lpcap -L$(STAGING_DIR)/usr/lib
+CFLAGS  += -DPPP_FILTER -I$(STAGING_DIR)/usr/include
 endif
 
 ifdef HAVE_INET6
